### ISSUE TITLE
Accessibility versus security of token expiry

### ISSUE THREAD

**Issue by jnywong** [2i2c-org/team-compass#860]
Capturing the comment to be addressed below:

> There's a balance between accessibility and security to be struck here, what do we think? I also think that the Grafana token should be stored as an organizational 2i2c secret since there are multiple uses, to avoid secrets proliferating. Happy to update the team compass with what we decide on this.

_Originally posted by @jnywong in https://github.com/2i2c-org/team-compass/pull/859#discussion_r1598414692_

**Comment by jnywong** [comment]
Team compass is updated with the following:

> To create a secret for the 2i2c organization for multiple users, see the [GitHub Docs – Using secrets in GitHub actions](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-an-organization). We recommend organization-level secrets against individual-level secrets to minimize the need to create duplicate secrets for multiple repositories.

https://team-compass.2i2c.org/reference/documentation/secrets/
